README.md

```diff
@@ -1,70 +1,74 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Todo Application
 
-## Available Scripts
+> This is an application that stores lists of books
 
-In the project directory, you can run:
 
-### `npm start`
+## Live Demo
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[Live demo]()
+## Built With
```

Comment:
- [ ] OPTIONAL - Please, consider fixing this broken link to the live demo version. You can remove it for this milestone 👍